### ISSUE TITLE
Incompatibility with ImageMagick installed via Homebrew and RMagick under MacRuby

### ISSUE THREAD

**Issue by VxJasonxV** [rmagick-temp/rmagick#49]
I'm having a heck of a time getting a few excellent pieces of software to work together.

I installed ImageMagick via homebrew. I even updated brew and re-installed it despite the only addition to ImageMagick being rsvg support.

ImageMagick Version:

```
Version: ImageMagick 6.7.5-7 2012-02-28 Q16 http://www.imagemagick.org
Copyright: Copyright (C) 1999-2012 ImageMagick Studio LLC
Features:  OpenCL
```

Then, via RVM, I installed MacRuby, created a gemset for my project and installed RMagick into it:

```
$ which ruby
/Users/jason/.rvm/rubies/macruby-0.10/bin/ruby

$ ruby --version
MacRuby 0.10 (ruby 1.9.2) [universal-darwin10.0, x86_64]

$ which gem
/Users/jason/.rvm/rubies/macruby-0.10/bin/gem

$ gem --version
1.4.2

$ rvm current
macruby-0.10@seb

$ gem list --local

*** LOCAL GEMS ***

json (1.6.5)
rmagick (2.13.1)
```

But when I go to test my installation, I get an RMagick2.so LoadError.

```
$ irb -rubygems -r RMagick
/Library/Frameworks/MacRuby.framework/Versions/0.10/usr/bin/macirb:20:in `block': no such file to load -- RMagick2.so (LoadError)
    from /Library/Frameworks/MacRuby.framework/Versions/0.10/usr/bin/macirb:31:in `block'
    from /Library/Frameworks/MacRuby.framework/Versions/0.10/usr/bin/macirb:9:in `block'
    from /Library/Frameworks/MacRuby.framework/Versions/0.10/usr/bin/macirb:7:in `<main>'
```

I suspect the problem is that since homebrew compiles all the items, and it was compiled by my version of OS X Lion with Xcode 4.3's toolchain, (llvm-gcc, I believe?), the default library output name is .bundle instead of .so

```
$ ls ~/.rvm/gems/macruby-0.10\@seb/gems/rmagick-2.13.1/lib/
RMagick.rb       RMagick2.bundle* rvg/
```

I assume there may be some kind of compiler options that I could use to force Homebrew to compile ImageMagick in the way I want, but (if I'm correct so far) ideally RVM would be able to use the .bundle file as well as the .so, for true cross platform support.

Am I close?

**Comment by VxJasonxV** [comment]
Actually, this might be an rvm issue?

```
$ which irb
/Users/jason/.rvm/rubies/macruby-0.10/bin/irb

$ irb -rubygems -r RMagick
/Library/Frameworks/MacRuby.framework/Versions/0.10/usr/bin/macirb:20:in `block': no such file to load -- RMagick2.so (LoadError)
    from /Library/Frameworks/MacRuby.framework/Versions/0.10/usr/bin/macirb:31:in `block'
    from /Library/Frameworks/MacRuby.framework/Versions/0.10/usr/bin/macirb:9:in `block'
    from /Library/Frameworks/MacRuby.framework/Versions/0.10/usr/bin/macirb:7:in `<main>'
```

I would think the rvm built version of MacRuby / irb would not be using the system version of MacRuby.

**Comment by Watson1978** [comment]
We fixed this issue with https://github.com/MacRuby/MacRuby/commit/9bc36edae700adc71f52ffe415e084242f13d931

However, its changing is not included in MacRuby 0.10.
You could use RMagick with MacRuby nightly.

Thanks!

**Comment by VxJasonxV** [comment]
I love rvm.

```
rvm install macruby-nightly
rvm use macruby-nightly
(redo all my gemset stuff too)
$ irb -rubygems -r RMagick
irb(main):001:0> quit
$
```

Thanks a ton Watson!
